### DISCUSSION TITLE
kitty: how to define only left option as alt

### DISCUSSION THREAD

**Discussion by emninger** [Homebrew/discussions#182] [Everyday usage]
My excuses in advance since this is not a properly homebrew related question, but my hope is some of capable coders here will know the answer to my question anyway.

In the kitty instructions there is said:

#: Use the option key as an alt key. With this set to no, kitty will
#: use the macOS native Option+Key = unicode character behavior. This
#: will break any Alt+key keyboard shortcuts in your terminal
#: programs, but you can use the macOS unicode input technique. You
#: can use the values: left, right, or both to use only the left,
#: right or both Option keys as Alt, instead.

Now, i do not see, what would be the correct syntax to define ONLY the left option key as alt. I tried in several ways but unfortunately to no extent.

Thanks a lot in advance for any pointer.

**Comment by SMillerDev** [comment]
I think the best way to get an answer is to ask the community for the Kitty software.

**Comment by carlocab** [comment]
Try reading this: https://sw.kovidgoyal.net/kitty/conf.html#opt-kitty.macos_option_as_alt

I think it just means you need to have `macos_option_as_alt left` in your config file.

If that doesn't work, have a look here to see if anyone else has had the same issue: https://github.com/kovidgoyal/kitty

**Reply by emninger** [reply]
Thank you! That's it.
